### PR TITLE
grafana/ui: add icon as a prefix in time range input component

### PR DESCRIPTION
**Pull request by reemtariqq**
**What is this feature?**

Adding icon as a prop to TimeRangeInput component

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

Anyone using grafana/ui 

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx [hunk 1]
```diff
@@ -31,6 +32,7 @@ export interface TimeRangeInputProps {
   /** Controls visibility of the preset time ranges (e.g. **Last 5 minutes**) in the picker menu */
   hideQuickRanges?: boolean;
   disabled?: boolean;
+  icon?: IconName;
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx:35]: I'm wondering if we want to allow to pass a custom icon or if we just want to use the same one we use in other time pickers? They use "clock-nine" and then we could have a `showIcon` prop. Does that work for you and what do you think about it? Just seems strange to allow to pass in a custom icon to a time picker...

**Review by JoaoSilvaGrafana — COMMENTED** [review]
Looks good otherwise! I think this will have to go into 10.0 now that we have frozen the 9.5 release though

File: packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx (hunk 1 above)
Comment (reemtariqq) [reply packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx:35]: make sense, will do the change

**Review by JoaoSilvaGrafana — APPROVED** [review]
nice 👍

**Pull request merged**